lonboard/_geoarrow/parse_wkb.py

```diff
@@ -54,28 +54,28 @@ def parse_wkb_table(table: pa.Table) -> List[pa.Table]:
     if GeometryType.LINEARRING in unique_type_ids:
         raise ValueError("LinearRings not currently supported")
 
-    point_indices = np.where(
-        (type_ids == GeometryType.POINT) | (type_ids == GeometryType.MULTIPOINT)
+    polygon_indices = np.where(
+        (type_ids == GeometryType.POLYGON) | (type_ids == GeometryType.MULTIPOLYGON)
```

Comment:
Can you revert these lines and 66-67, since these aren't connected to the ordering?

Reply:
Sure thing, seems reasonable